src/main/java/Marites.java

```diff
@@ -0,0 +1,198 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+import static java.lang.Integer.parseInt;
+
+public class Marites {
+
+    /* STRING CONSTANTS */
+
+    // Logo found in https://emojicombos.com/kaomoji
+    private static final String INTRO_MESSAGE = "(งツ)ว\n" +
+            "Hi, I'm Marites! I've heard so many things about you!\n" +
+            "I have a lot of stories to share, but first, how can I help you?\n";
+    private static final String EXIT_MESSAGE = "See you next time!";
```

Comment:
May not need constants for these messages as the method names are clear on what they are.